tgui/.yarnrc.yml

```diff
@@ -1,8 +1,8 @@
+enableScripts: false
+
 logFilters:
-  ## MUST_BUILD
-  ## It's an opencollective advert in postinstall that is not even displayed
-  ## correctly in Yarn, who the fuck cares.
-  - code: YN0007
+  ## DISABLED_BUILD_SCRIPTS
+  - code: YN0004
     level: discard
   ## FETCH_NOT_CACHED
   ## Too many cache misses on first install.
```

Comment:
You might be interested in [`preferAggregateCacheInfo`](https://yarnpkg.com/configuration/yarnrc#preferAggregateCacheInfo) which will make Yarn print a summary instead

Reply:
Cool, didn't know that, will try this evening.